apps/web-giddh/src/app/vat-report/vatReport.component.html

```diff
@@ -13,226 +16,195 @@
                                     <div class="tax-heading col-md-3 col-sm-3 col-12">
                                         <label>{{commonLocaleData?.entity}}:</label>
                                     </div>
-                                    <div class="tax-dropdown col-md-9 col-sm-9 col-12 linked-entities">
```

Comment:
@shalinee-walkover please check this code is getting removed

Reply:
added, Done